docs/integrations/amazon-aws/lambda.md

```diff
@@ -193,7 +193,7 @@ AWS Lambda provides Provisoned Concurrency for greater control over the start up
 
 To collect these metrics in Sumo Logic, do the following:
 
-1. Complete [Step](#Collect-Logs-for-the-AWS-Lambda-App#collect-amazon-cloudwatch-metrics).
+1. Complete [Step](#collect-amazon-cloudwatch-metrics).
```

Comment:
There should be a number for the step.

Reply:
There is no number to the step, as it's a sub-section within the doc. I have added an article for the sentence to be complete.